PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetExtractor.h

```diff
@@ -176,7 +176,7 @@ class AliAnalysisTaskJetExtractor : public AliAnalysisTaskEmcalJet {
   AliAnalysisTaskJetExtractor &operator=(const AliAnalysisTaskJetExtractor&); // not implemented
 
   /// \cond CLASSIMP
-  ClassDef(AliAnalysisTaskJetExtractor, 12) // Jet extraction task
+  ClassDef(AliAnalysisTaskJetExtractor, 13) // Jet extraction task
```

Comment:
Just fyi, since you didn't change any streaming members, you don't need to update ClassDef here. But it's also not a problem